model-optimizer/extensions/back/AvgPool.py

```diff
@@ -20,5 +20,6 @@ def pattern(self):
         )
 
     def replace_pattern(self, graph: Graph, match: dict):
-        match['pooling'].type = 'AvgPool'
-        del match['pooling']['pool_method']
+        node = match['pooling']
+        node.type = 'AvgPool'
```

Comment:
use node.soft_get('type')

Reply:
I am setting the value of the attribute `type`, no need to use `soft_get`